packages/ui-tests/src/coreComponents/Menu.spec.ts

```diff
@@ -25,6 +25,38 @@ context('Menu', () => {
     cy.get(popup).should('not.exist')
   })
 
+  it('Should close the menu when clicking on the button again', () => {
+    cy.get(menu1).eq(0).find('button').click()
+    cy.get(popup).eq(0).should('exist').should('be.visible')
+
+    cy.get(menu1).eq(0).find('button').click()
+    cy.get(popup).should('not.exist')
+  })
+
+  it('Should close the menu when clicking on the button again', () => {
```

Comment:
Why do you need the same test again, see line 28?

Reply:
Removed